server/src/main/java/org/elasticsearch/index/mapper/ParametrizedFieldMapper.java

```diff
@@ -570,6 +570,9 @@ public final void parse(String name, ParserContext parserContext, Map<String, Ob
                     throw new MapperParsingException("unknown parameter [" + propName
                         + "] on mapper [" + name + "] of type [" + type + "]");
                 }
+                if (Objects.equals("boost", propName)) {
+                    deprecationLogger.deprecate("boost", "Parameter [boost] on field [{}] is deprecated", name);
```

Comment:
maybe add that it will be removed in 8 since that's the new plan ?

Reply:
++